README.md

```diff
@@ -530,12 +525,9 @@ The result would then not be obfuscated.
 
 ### Soft Fail
 
-Using the soft fail feature will not fail the CI/CD step within the pipeline if the Cycode scan detects an issue.
-If an issue occurs during the Cycode scan, using a soft fail feature will automatically execute with success (`0`) to avoid interference.
-
-To configure this feature, add the `--soft-fail` option to any type of scan. This will force the scan results to succeed (exit code `0`).
+In normal operation the CLI will return an exit code of `1` when issues are found in the scan results. Depending on your CI/CD setup this will usually result in an overall failure. However, if you dont want this to happen you can use the soft fail feature.
```

Comment:
```suggestion
In normal operation the CLI will return an exit code of `1` when issues are found in the scan results. Depending on your CI/CD setup this will usually result in an overall failure. If you don't want this to happen, you can use the soft fail feature.
```